src/pages/Validators/Index.tsx

```diff
@@ -54,7 +53,15 @@ export default function ValidatorsPage() {
         },
       );
     }
-  }, []);
```

Comment:
might be another case where we want to trigger the event only once / on initial mount? not sure though without digging deeper into the logic